source/kubernetes/versions.rst

```diff
@@ -63,15 +63,39 @@ It does not show patch versions, as all patch versions for a supported minor ver
      - 2024-05-27
      - Expected 2025-02-28
    * - ``1.30``
-     - In development
-     - Expected 2024-09-02
+     - Supported
+     - 2024-08-26
      - Expected 2025-06-28
+   * - ``1.31``
+     - In development
+     - Expected 2024-10-31
+     - Expected 2025-10-28
 
 
 **********************
 Version upgrade notes
 **********************
 
+Version v1.29 to v1.30
+======================
+
+Kubernetes `release changelog for v1.30 since v1.29`_.
+
+.. _`release changelog for v1.30 since v1.29`: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.30.md
+
+In addition to the Kubernetes changes, we have:
+
+* Upgraded minor version of Calico CNI.
+* Upgraded minor version of Cinder CSI and Cloud Provider Openstack.
+
+
+Note that skipping minor versions when upgrading a cluster is unsupported and
+will result in a broken cluster. Please refer to `official documentation`_ on
+upgrading kubeadm clusters.
+
+.. _`official documentation`: https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade
```

Comment:
This is a little too 'behind the scenes' for our customer facing docs. While we don't hide the technology used, we don't need to refer to `kubeadm` implementation docs.

Perhaps we just change this to "For example, before upgrading to `v1.30.x`, you must have upgraded to `v1.29.x`"
Or link to https://docs.catalystcloud.nz/kubernetes/cluster-upgrade.html#identify-the-cluster-template-to-upgrade-to